web/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, routes } from '@redwoodjs/router'
+import { MetaTags } from '@redwoodjs/web'
+
+import Button from 'src/components/Button/Button';
+
+import LandingBG from '../../../public/landingBg.svg';
+import WelcomeIMG from '../../../public/welcomeImg.svg';
+
+const LandingPage = () => {
+  return (
+    <>
+      <MetaTags title="Landing" description="Landing page" />
+
+      <div className='relative w-full'>
+      <div className='max-h-screen overflow-hidden z-[-1]'>
```

Comment:
Should this be `min-h-screen` so that it would take up the entire screen on bigger displays?